src/sentry/templates/sentry/emails/recover_account.html

```diff
@@ -7,6 +7,10 @@
     <p>A password reset was requested for your account ({{ user.username|safe }}) on Sentry ({{ domain }}). If you did not authorize this, you may simply ignore this email.</p>
     <p>To continue with your password reset, simply click the button below, and you will be able to change your password. This link will expire in 48 hours.</p>
 
-    <a href="{{ url|safe }}" class="btn">Reset Password</a>
-</p>
+    <p><a href="{{ url|safe }}" class="btn">Reset Password</a></p>
+
+    <p><small>This request was made by:</small></p>
+    <p><small><pre>
+IP: {{ ip_address }}
+Date: {{ datetime }} UTC</pre></small><p>
```

Comment:
Wouldn't it make sense to grab the timezone from the [`date` filter](https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#date) instead of specifying it statically here?